packages/devtools/src/components/ActivityDetails/ActivityDetails.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Button,
+  Tooltip,
+  Switch,
+  InfoLabel,
+  useToastController,
+  Toast,
+  ToastTitle,
+} from '@fluentui/react-components';
+import { CopyRegular } from '@fluentui/react-icons/lib/fonts';
+import Json from '../Json/Json';
+import useActivityDetailsClasses from './ActivityDetails.styles';
+import { ActivityEvent } from '../../types/Event';
+
+interface ActivityDetailsProps {
+  selected: ActivityEvent;
+  view: 'preview' | 'json';
+  setView: (view: 'preview' | 'json') => void;
+}
+
+const ActivityDetails: React.FC<ActivityDetailsProps> = ({ selected, view, setView }) => {
+  const classes = useActivityDetailsClasses();
+  const { dispatchToast } = useToastController();
+
+  const handleCopy = async () => {
+    try {
+      await navigator.clipboard.writeText(JSON.stringify(selected));
+
+      // Show toast notification
+      dispatchToast(
+        <Toast>
+          <ToastTitle role="status" aria-live="polite">
+            Content copied to clipboard
+          </ToastTitle>
+        </Toast>,
+        { position: 'bottom', timeout: 2000 }
+      );
+    } catch (err) {
+      console.error('Failed to copy:', err);
```

Comment:
we should try to use the `@teams.sdk/common/logging` `ConsoleLogger` to do this logging when possible.